requirements.txt

```diff
@@ -1,4 +1,4 @@
-antlr4-python3-runtime==4.7.2
+antlr4-python3-runtime==4.8
```

Comment:
Good catch! :100: Wondering: could it make sense to remove this requirement? :thinking: That way blackbird would the correct version automatically.

Reply:
(For lack of a better place to comment:) Think we missed the pinning of `quantum-blackbird>=0.3.0` in the `setup.py` in #476

Reply:
:thinking: True, I'm also thinking we could remove it from here, since it's already a requirement in Blackbird (and it doesn't seem to be used by SF at all). I wonder why it was added here at all though.

 Re using Blackbird >= 0.3, good catch. I'll update this PR to handle both these changes.